pandas/core/indexing.py

```diff
@@ -449,6 +449,38 @@ def loc(self) -> _LocIndexer:
         8          4       5
         9          7       8
 
+        **Set values with a Series**
```

Comment:
this is ok but really not very idiomatic e.g. these should be on the ```__setitem__``` doc-string e.g.
```df['shield'] = ```

Reply:
Looks like this is the docstring of `.loc`, that's why is using this idiom.

Reply:
sure but these are not idiomatic patterns for *setting a column*; sure if you have a *mask* then its the idimoatic way, but NOT for a column.